src/test/suite/fixtures/recorded/languages/php/chuckType4.yml

```diff
@@ -0,0 +1,29 @@
+languageId: php
+command:
+  version: 1
+  spokenForm: chuck type
+  action: remove
+  targets:
+    - type: primitive
+      modifier: {type: containingScope, scopeType: type, includeSiblings: false}
+initialState:
+  documentContents: |-
+    <?php
+
+    (string) $str;
+  selections:
+    - anchor: {line: 2, character: 10}
+      active: {line: 2, character: 10}
+  marks: {}
+finalState:
+  documentContents: |-
+    <?php
+
+    () $str;
```

Comment:
These parens should be checked as well

Reply:
Yeah that makes sense to me.  I think you'll need a bit of custom code.  First, you'll want to back away from `*[type]` for this case, so you could change it to `~cast_expression[type]`.  Then, you'll create a separate matcher for `cast_expression`, using a custom extractor, similar to the one we use for `name` in Markdown https://github.com/cursorless-dev/cursorless-vscode/blob/cfbfcb116828cd98b9cb0723a82595f5a212e0ec/src/languages/markdown.ts#L29-L45.  You'll return the range of the `node.childForFieldName("type")` for the content range, and the node itself for the `outerSelection`.  

Lmk if this doesn't make sense and we can do a quick pairing session

Reply:
Thank you for the direction, that was exactly what I needed 👍